hippunfold/workflow/rules/common.smk

```diff
@@ -24,41 +24,69 @@ def get_modality_suffix(modality):
 
 
 def get_final_spec():
-    if len(config["hemi"]) == 2:
-        specs = expand(
+    specs = expand(
+        bids(
+            root=root,
+            datatype="surf",
+            den="{density}",
+            space="{space}",
+            hemi="{hemi}",
+            label="{autotop}",
+            suffix="surfaces.spec",
```

Comment:
previously the surfaces spec would have both hemispheres in it - this seems to change it so we have a spec file for each hemisphere? was that intended?